.github/workflows/deploy-metadata.yml

```diff
@@ -1,6 +1,6 @@
 # Flow out an environment (sync metadata from Git to Salesforce)
 
-name: Flow out an environment
+name: Deploy metadata
```

Comment:
I don't think this is more descriptive. Outbound flow is a lot different from a mere deployment. At the very least I think it should say "deploy metadata changes", but I would propose the following two names for inbound and outbound workflows:
- Commit metadata changes from org
- Deploy metadata changes to org